Common/CommonFuncs.h

```diff
@@ -28,11 +30,12 @@
 #include <unistd.h>
 #include <errno.h>
 
-#if defined(_M_IX86) || defined(_M_X86)
+#if PPSSPP_ARCH(X86) || PPSSPP_ARCH(AMD64)
 #define Crash() {asm ("int $3");}
-#else
-#include <signal.h>
-#define Crash() {kill(getpid(), SIGINT);}
```

Comment:
I think that leaving this as a default fallback in an `#else` at the end might help ports to other platform such as POWER or RISC-V.

Reply:
Yeah, that's a fair point.